gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -0,0 +1,368 @@
+<path-to-registration-file>
+----
+====
+
+. Replace `<path-to-registration-file>` with the path and filename of the existing `registration.yaml` file. You can optionally output the new registration file to a directory by using the `--output-directory` flag. The `renew` command overwrites the existing registration file if you specify the file's directory as the output directory.
+
+. Replace <your-username> and <your-password> with Anypoint Platform user credentials.
+
+. Run the command.
++
+The output should include the message:
++
+[source,text]
+----
+Registration renewal completed
+----
+
+[[renew-with-conn-app]]
+== Renew Registration with Connected App Credentials
+
+. If you don't have a username or password for Anypoint Platform, you can xref:access-management::connected-apps-developers.adoc#create-a-connected-app[configure a connected app] via Anypoint Access Management.
+.. Include the following scopes:
+
+* Read Servers
+* Manage Servers
+* View Organization
+
+.. Save the *Id* and *Secret* of the connected app you configure, then use these credentials in the `renew` command.
+
+. Create a new directory called `flex-renew-registration` (or similar). You run the `renew` command in this new directory.
+
+. Use the following command:
++
+[tabs]
+====
+Linux::
++
+[source,ssh]
+----
+flexctl registration renew --client-id=<your-client-id> --client-secret=<your-client-secret> \
+ <path-to-registration-file>
+----
++
+Use `sudo` if you encounter file permission issues when running this command.
+
+Docker::
++
+For Flex Gateway running in a container, invoke `renew` as part of the container `run` command.
++
+[source,ssh]
+----
+docker run --entrypoint flexctl \
+-v "$(pwd)":/renew \
+-v <path-to-registration-directory>:/registration \
+-u $UID mulesoft/flex-gateway \
+registration renew \
+--client-id=<your-client-id> \
+--client-secret=<your-client-secret> \
+--output-directory=/renew \
+<path-to-registration-file>
+----
+
+Podman::
++
+For Flex Gateway running in a container, invoke `renew` as part of the container `run` command.
++
+[source,ssh]
+----
+podman run --entrypoint flexctl --userns=keep-id \
+-v "$(pwd)":/renew:Z \
+-v <path-to-registration-directory>:/registration:Z \
+-u $UID docker.io/mulesoft/flex-gateway \
+registration renew \
+--client-id=<your-client-id> \
+--client-secret=<your-client-secret> \
+--output-directory=/renew \
+<path-to-registration-file>
+----
+====
+
+. Replace `<path-to-registration-file>` with the path and filename of the existing `registration.yaml` file. You can optionally output the new registration file to a directory by using the `--output-directory` flag. The `renew` command overwrites the existing registration file if you specify the file's directory as the output directory.
+
+. Replace <your-client-id> and <your-client-secret> with the connected app credentials.
+
+. Run the command.
++
+The output should include the message:
++
+[source,text]
+----
+Registration renewal completed
```

Comment:
this is how the final message looks like

`Starting registration renewal, please be patient.
Registration renewal completed, new expiration date is 2025-09-25 19:27:32 +0000 UTC. Configuration files were written in directory "./resources/examples/common/config.d". For security, modify the file permissions to restrict production scenario access to the user running flex.`